cluster/local/kind.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+set -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# shellcheck disable=SC1090
+projectdir="${scriptdir}/../.."
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${scriptdir}/../.. build.vars)
+
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-amd64"
+MINIKUBE_IMAGE="${DOCKER_REGISTRY}/${PROJECT_NAME}:master"
```

Comment:
Not a functional change, but kind of confusing that this is called MINIKUBE_IMAGE